eth/chains/base.py

```diff
@@ -577,13 +585,23 @@ def import_block(self, block: BaseBlock, perform_validation: bool=True) -> BaseB
             ensure_imported_block_unchanged(imported_block, block)
             self.validate_block(imported_block)
 
-        self.chaindb.persist_block(imported_block)
+        new_canonical_headers, old_headers = self.chaindb.persist_block(imported_block)
```

Comment:
Can we use the suffix `_hashes` on these as to not accidentally confuse them to be `BlockHeader` objects.

- `new_canonical_header_hashes`
- `old_header_hashes`

Also, maybe instead of the term `old` we could use one of

- `orphaned`
- `abandoned`
- `discarded`

cc @carver as he normally has thoughts about the naming of things.

Reply:
:+1: to hashes. All of the `old` alternatives are pretty good. I think `orphaned` is my favorite: it's used pretty regularly for exactly this type of header/block: no longer on the longest/main chain.